client/src/models/Filter.test.js

```diff
@@ -18,16 +18,16 @@ describe('Filter, with initial values about this filter object', () => {
 
 	it('getBackloopString() should be: ', () => {
 		console.log(filter.getBackloopString())
-		expect(filter.getBackloopString().indexOf('&filter[where][id]=10') >= 0).toBe(true)
-		expect(filter.getBackloopString().indexOf('&filter[where][status]=planted') >=0).toBe(true)
-		expect(filter.getBackloopString().indexOf('&filter[where][timeCreated][between]=2019-07-25&filter[where][timeCreated][between]=2019-07-30') >=0).toBe(true)
+		expect(filter.getBackloopString()).toEqual(expect.stringContaining('&filter[where][id]=10'))
+		expect(filter.getBackloopString()).toEqual(expect.not.stringContaining('&filter[where][status]=planted'))
```

Comment:
This looks like the opposite of the original test (which I'm guessing failed because `'Planted' !== 'planted'`).

Reply:
the Filter class has the status option removed from it, so it doesn't generate status query anymore. This is per @dadiorchen I think, so I asked and he said it was ok to pull it from the test. Rather than pull it, I decided to do a negative match.